### PR TITLE
24 profile icon

### PR DESCRIPTION
**Pull request by Rahul-Anantuni**

### CODE REVIEW

File: react-app/src/App.tsx
Comment (Rahul-Anantuni) [inline react-app/src/App.tsx:1]: I didn't mean to change anything in App.tsx. Please ignore anything here

File: react-app/src/components/ProfileIcon/ProfileIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+interface ProfileIconProps {
+  user: Volunteer;
+}
+
+const ProfileIcon: React.FC<ProfileIconProps> = ({ user }) => {
+  const initials = `${user.firstName.charAt(0)}${user.lastName.charAt(0)}`;
+
+  return (
+    <div
```
Comment (akashpatil0) [inline react-app/src/components/ProfileIcon/ProfileIcon.tsx]: Move all the styling for this page to its own CSS file.

File: react-app/src/components/ProfileIcon/ProfileIcon.tsx [hunk 2]
```diff
@@ -14,12 +14,13 @@ const ProfileIcon: React.FC<ProfileIconProps> = ({ user }) => {
         width: "50px",
         height: "50px",
         borderRadius: "50%",
-        backgroundColor: "forestgreen",
+        backgroundColor: "#0A7650",
```
Comment (akashpatil0) [inline react-app/src/components/ProfileIcon/ProfileIcon.tsx]: This should use the `--forest-green` color as described in the task description.

Comment (Rahul-Anantuni) [reply react-app/src/components/ProfileIcon/ProfileIcon.tsx]: Fixed.

File: react-app/src/components/ProfileIcon/ProfileIcon.tsx (hunk 1 above)
Comment (Rahul-Anantuni) [reply react-app/src/components/ProfileIcon/ProfileIcon.tsx]: Fixed.

**Review by sophtsai — APPROVED** [review]
Looks good! I just refactored the profile icon to remove the parameter since you should be able to access the (same) user directly from the component.

**Pull request merged**